osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -216,7 +216,18 @@ private Drawable createDrawablePerformance()
             if (!Score.PP.HasValue)
             {
                 if (Score.Beatmap?.Status.GrantsPerformancePoints() == true)
+                {
+                    if (!Score.Ranked)
```

Comment:
Three nested ifs is the point where I call enough. Can you refactor this so that this doesn't happen? Rider tells me you can invert just ifs twice.